tests/IsoCodes/Tests/SsnTest.php

```diff
@@ -77,15 +136,35 @@ public function testValidSsn($ssn)
      */
     public function testInvalidSsn($ssn)
     {
-        $this->assertFalse($this->ssn->validate($ssn));
```

Comment:
AFAIK, only major version can include incompatible API changes. Please do not change existing unit tests until the next major release.

Unit test are for regression evaluations, not only code-covering fancy and green badges races. You cannot change existing unit tests just to impose something new.

Real library users are your customers: Until next major release you cannot force them to change their code just because you want to impose a new implementation to serve your projects.

IMHO, in a non-major release asserting that a PR will not breaking compatibility for projects at all, unless maybe unit testing is a misunderstanding of the unit testing concept and what regression means (and costs) for your users.

Please be patient until the next major release, we'll certainly find a way to introduce your changes at that moment.